client/src/components/layout/AccountPage.js

```diff
@@ -0,0 +1,67 @@
+import React, { useState, useEffect } from "react";
+import _ from "lodash";
+import Dropzone from "react-dropzone";
+
+const AccountPage = (props) => {
+  const [user, setUser] = useState(props.user);
+  const handleImageUpload = (acceptedImage) => {
+    setUser({ ...user, image: acceptedImage[0] });
+  };
+
+  const addProfilePicture = async () => {
+    const newProfilePictureBody = new FormData();
+    newProfilePictureBody.append("image", user.image);
+    try {
+      const response = await fetch(`/api/v1/users/${user.id}`, {
+        method: "PATCH",
+        headers: {
+          Accept: "image/jpeg",
+        },
+        body: newProfilePictureBody,
+      });
+      if (!response.ok) {
+        throw new Error(`${response.status} (${response.statusText})`);
+      }
+      const body = await response.json();
+      setUser({ ...user, image: body.user.image });
```

Comment:
this looks good - once the image is uploaded, we update the "source of truth" user state to display the now saved image on the page